packages/contracts/deploy/new/30_plugins/00_plugin-setups/21_admin_setup_conclude.ts

```diff
@@ -34,3 +35,4 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
 export default func;
 func.tags = ['New', 'AdminSetup', 'Verify'];
+func.skip = async hre => await skipIfZkSync(hre, 'AdminSetupConclude');
```

Comment:
Is the admin plugin not going to be deployed on zksync?

Reply:
Nope, it was also using OZ's clone which is not supported on zksync. So We decided to not deploy Admin instead of coming up with another AdminZksync as this plugin is not at all important.